02 - Community Expansions/02.01 Plugins by Category/View.md

```diff
@@ -9,17 +9,19 @@ publish: true
 
 # View
 
-Change views and display
+Plugins that change or add new views, and modify how notes are displayed.
 
 ## Plugins in this category
 
 - [[sliding-panes-obsidian|Sliding Panes (Andy Matuschak Mode)]]
+- [[obsidian-tabs|Obsidian Tabs]]
+
```

Comment:
Suggest removing blank lines here, to remove blank line between every bullet in the list, in Preview